airbyte-ci/connectors/cdk/cdk/java_cdk/tasks.py

```diff
@@ -0,0 +1,109 @@
+"""CI Workflow tasks for Airbyte Java CDK"""
+
+import os
+
+import pygit2
+from aircmd.actions.environments import with_gradle
+from aircmd.models.base import PipelineContext
+from aircmd.models.utils import load_settings, make_pass_decorator
+from dagger import Client, Container
+from prefect import task
+from prefect.artifacts import create_link_artifact
+
+from .settings import JavaCDKSettings
+
+pass_pipeline_context = make_pass_decorator(PipelineContext, ensure=True)
+pass_global_settings = make_pass_decorator(JavaCDKSettings, ensure=True)
+
+
+@task
+async def build_java_cdk_task(client: Client, settings: JavaCDKSettings, ctx: PipelineContext, scan: bool = False) -> Container:
+    files_from_host = [
+        "./build.gradle",
+        "./gradlew",
+        "./gradle",
```

Comment:
It's because I'm running this from a different directory than the root dir, which is where with_gradle expects. I will clean this up as part of the pr